Cargo.toml

```diff
@@ -16,18 +16,18 @@ exclude = [
 ]
 
 [badges]
-travis-ci = { repository = "dnaq/sodiumoxide" }
+travis-ci = { repository = "sodiumoxide/sodiumoxide" }
 
 [dependencies]
-libc = "0.2"
+libc = { version = "^0.2.41" , default-features = false }
```

Comment:
you don't have to mention `^`
`x.y.z` = `^x.y.z`

Reply:
I didn't realize that was the default operator. Now I know.

I was reading https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html and I hadn't seen it before, but like the way the tilde operator works. I think that its use (i.e., specifying `"~x.y.z"`) could minimize breakage and also force us to be more intentional (e.g., try to review upstream changes) when upgrading minor versions of the (very few) dependencies of this crate. Thoughts?

Side note: an @alias for the sodiumoxide maintainers would be useful.

Reply:
~ is good when you don't want to limit it to the a specific minor dependency, but I don't see the case for it here. most crates follow semver well enough to be backwards compatible. So x.y.z should be fine

Reply:
Fair enough. I'm going to leave this as-is then since the only "problem" is it's slightly verbose syntax.